libs/orders/src/lib/order-hooks/use-order-submit.tsx

```diff
@@ -13,7 +13,7 @@ import { Icon, Intent } from '@vegaprotocol/ui-toolkit';
 import { t } from '@vegaprotocol/i18n';
 import type { OrderSubmissionBody } from '@vegaprotocol/wallet';
 
-export const getOrderDialogTitle = (
+export const getOrderTitle = (
```

Comment:
chore (blocking): This file was a dead file. Can we 

a) move these helper functions to libs/orders/src/lib/utils.ts
b) rename to getOrderToastTitle etc etc
c) delete this file and corresponding test file